debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -340,7 +340,7 @@ public void initializeStorage() {
 
             // Create topic
             final NewTopic topic = new NewTopic(topicName, (short)1, replicationFactor);
-            topic.configs(Collect.hashMapOf("cleanup.policy", "delete", "retention.ms", Long.toString(Long.MAX_VALUE)));
+            topic.configs(Collect.hashMapOf("cleanup.policy", "delete", "retention.ms", Long.toString(Long.MAX_VALUE), "retention.bytes", "-1"));
```

Comment:
-1 is the default as per the [docs](https://kafka.apache.org/documentation/). Is this to protect against the case where a global default is given (if that's even doable)?

Reply:
Ok, so IIUC, if there's a server default set for `retention.bytes`, it'll apply to this topic as we don't override the value when creating the topic. That's bad. In fact, I think this warrants an out-of-order release next Monday.

Reply:
Exactly, this is to cover the cases where default was overridden.